srcpkgs/src/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'src'
+pkgname=src
+version=1.37
+revision=1
+build_style=gnu-makefile
+hostmakedepends="python3"
```

Comment:
```suggestion
hostmakedepends="python3 rsc"
```

It tries to execute `rcs --version 2>/dev/null` in build and check stages. The check error persists though.

Reply:
Wouldn't it be checkdepends?

Reply:
> Wouldn't it be checkdepends?

It is used in the build stage also.

Reply:
ok thanks